config.defaults.yaml

```diff
@@ -51,3 +162,137 @@ docs:
       coordinates: [254.7213, 32.8341]
       period: 0.31087445597147706
       title: RR Lyrae c
+
+features:
+  phenomenological:
+    - ra
+    - dec
+#    - n
+    - median
+    - wmean
+    - chi2red
+    - roms
+    - wstd
+    - norm_peak_to_peak_amp
+    - norm_excess_var
+    - median_abs_dev
+    - iqr
+    - i60r
+    - i70r
+    - i80r
+    - i90r
+    - skew
+    - smallkurt
+    - inv_vonneumannratio
+    - welch_i
+    - stetson_j
+    - stetson_k
+    - ad
+    - sw
+    - n_ztf_alerts
+    - mean_ztf_alert_braai
+    - period
+    - significance
+    - pdot
+    - f1_power
+    - f1_BIC
+    - f1_a
+    - f1_b
+    - f1_amp
+    - f1_phi0
+    - f1_relamp1
+    - f1_relphi1
+    - f1_relamp2
+    - f1_relphi2
+    - f1_relamp3
+    - f1_relphi3
+    - f1_relamp4
+    - f1_relphi4
+  ontological:
+    - ra
+    - dec
+    - median
+    - wmean
+    - chi2red
+    - roms
+    - wstd
+    - norm_peak_to_peak_amp
+    - norm_excess_var
+    - median_abs_dev
+    - iqr
+    - i60r
+    - i70r
+    - i80r
+    - i90r
+    - skew
+    - smallkurt
+    - inv_vonneumannratio
+    - welch_i
+    - stetson_j
+    - stetson_k
+    - ad
+    - sw
+    - n_ztf_alerts
+    - mean_ztf_alert_braai
+    - period
+    - significance
+    - pdot
+    - f1_power
+    - f1_BIC
+    - f1_a
+    - f1_b
+    - f1_amp
+    - f1_phi0
+    - f1_relamp1
+    - f1_relphi1
+    - f1_relamp2
+    - f1_relphi2
+    - f1_relamp3
+    - f1_relphi3
+    - f1_relamp4
+    - f1_relphi4
+    - AllWISE__w1mpro
+    - AllWISE__w1sigmpro
+    - AllWISE__w2mpro
+    - AllWISE__w2sigmpro
+    - AllWISE__w3mpro
+    - AllWISE__w3sigmpro
+    - AllWISE__w4mpro
+    - AllWISE__w4sigmpro,
+#    - AllWISE__ph_qual,
+    - Gaia_EDR3__phot_g_mean_mag
+    - Gaia_EDR3__phot_bp_mean_mag
+    - Gaia_EDR3__phot_rp_mean_mag
+    - Gaia_EDR3__parallax
+    - Gaia_EDR3__parallax_error
+    - Gaia_EDR3__pmra
+    - Gaia_EDR3__pmra_error,
+    - Gaia_EDR3__pmdec
+    - Gaia_EDR3__pmdec_error
+    - Gaia_EDR3__astrometric_excess_noise
+    - Gaia_EDR3__phot_bp_rp_excess_factor
+    - PS1_DR1__gMeanPSFMag
+    - PS1_DR1__gMeanPSFMagErr
+    - PS1_DR1__rMeanPSFMag
+    - PS1_DR1__rMeanPSFMagErr
+    - PS1_DR1__iMeanPSFMag
+    - PS1_DR1__iMeanPSFMagErr
+    - PS1_DR1__zMeanPSFMag
+    - PS1_DR1__zMeanPSFMagErr
+    - PS1_DR1__yMeanPSFMag
+    - PS1_DR1__yMeanPSFMagErr
+
+feature_norms:
+
+training:
+  dataset: md5 checksum? filename? from GCS? to use for training
+  classes:
+    agn:
+      label: AGN
+      # value should refer to features section of this config
+      features: ontological
+      # our "labels" are floats [0., 0.25, 0.5, 0.75, 1.]
+      threshold: 0.7
+      # balance ration for the prevalent class. leave as null to use all available data
```

Comment:
"ratio". Where do we store numbers for threshold and balance used in the classifier versions so far?

Reply:
This has been a major issue with how things were set up. One would need to dig trough quite a bit of logs to extract that info. My plan has been: once this PR is merged, incorporate [W&B](wandb.com) support to track all the data/metadata/artifacts. There are other options of course, @stefanv and I have been thinking about that - this is something that we should discuss.

Reply:
The main point of this PR is to make things easily "versionable"/trackable so that it only a matter of choosing the tool to do the tracking.